editor/src/components/inspector/sections/style-section/background-subsection/background-picker.tsx

```diff
@@ -466,6 +476,31 @@ export const BackgroundPicker: React.FunctionComponent<
     [stops, onSubmitValueAndUpdateLocalStops, selectedStopUnorderedIndex],
   )
 
+  const closePopup = props.closePopup
+
+  const dispatchEyeDropper = React.useCallback(() => {
+    const selectedViews = selectedViewsFromStore
+    if (selectedViews.length === 0) {
+      return
+    }
+    const EyeDropper = window.EyeDropper
```

Comment:
This segment looks a bit like we could wrap it up in a little function as it's the same in both files changed in this PR.

Reply:
It does look like it would work well as a `useEyeDropper`

Reply:
I created a utility function in https://github.com/concrete-utopia/utopia/pull/3115/commits/79100a9a4b198caa19167298c517d86bccf8c5df, we can put it into a use if this turns out to be a feature that's used in a lot of places